### PR TITLE
Update SonarQube 8.1 and 7.9.2

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: library/sonarqube [hunk 1]
```diff
@@ -4,17 +4,17 @@ Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
...
-Tags: 7.9.1-community, 7.9-community, latest, lts
+Tags: 7.9.2-community, 7.9-community, latest, lts
 Directory: 7/community
 
-Tags: 8.0-community-beta, 8-community-beta, community-beta
+Tags: 8.1-community-beta, 8-community-beta, community-beta
 Directory: 8/community
 
-Tags: 8.0-developer-beta, 8-developer-beta, developer-beta
+Tags: 8.1-community-beta, 8-developer-beta, developer-beta
```
Comment (jeff-zapotoczny-sonarsource) [inline library/sonarqube]: Shouldn't this tag be "8.1-developer-beta" and not "community"?

File: library/sonarqube [hunk 2]
```diff
@@ -4,17 +4,17 @@ Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
...
 Directory: 8/developer
 
-Tags: 8.0-enterprise-beta, 8-enterprise-beta, enterprise-beta
+Tags: 8.1-community-beta, 8-enterprise-beta, enterprise-beta
```
Comment (jeff-zapotoczny-sonarsource) [inline library/sonarqube]: Like above, I think you accidentally pasted "community" instead of "enterprise"

File: library/sonarqube (hunk 1 above)
Comment (ast3150) [reply library/sonarqube]: I think this is fixed in a newer PR #7159

**Comment by ast3150** [comment]
@jeff-zapotoczny-sonarsource I think this is fixed in a newer PR #7159

**Pull request closed without merging**